examples/swarm/image/docker/Dockerfile.template

```diff
@@ -0,0 +1,9 @@
+FROM postgres:${PGVERSION}
```

Comment:
Isn't this template the same as the one in `examples/kubernetes/image`?

Reply:
Yes indeed. The content of both `examples/kubernetes/image` and `examples/swarm/image` are the same. My goal with this PR was to update the swarm example base on the kubernetes example. So I chose to use the same docker images for both examples.

Reply:
@Nowheresly I'd like to avoid maintaining multiple set of identical files. I think we could remove these files and update the documentation to point to the examples/kubernetes/image.